### PR TITLE
[Testing] Update database.rst

### PR DESCRIPTION
**Pull request by JeroenMoonen**
Updated typo

### CODE REVIEW

File: testing/database.rst [hunk 1]
```diff
@@ -98,7 +98,7 @@ so, get the entity manager via the service container as follows::
 
     class ProductRepositoryTest extends KernelTestCase
     {
-        private \Doctrine\ORM\EntityManager $entityManager;
+        private \Doctrine\Persistence\ObjectManager $entityManager;
```
Comment (OskarStark) [inline testing/database.rst:101]: Please import the class

**Comment by xabbuh** [comment]
Can you explain why you think that this change is necessary?

File: testing/database.rst (hunk 1 above)
Comment (javiereguiluz) [reply testing/database.rst:101]: I've imported the class while merging.

**Comment by javiereguiluz** [comment]
@xabbuh this article is one of the few in Symfony Docs that uses `ObjectManager` (it uses it in the example above the change made here). So, for consistency, let's use it in all the examples of this page.

In a separate PR, we might want to change all `ObjectManager` examples in docs (there aren't many) in favor of using `EntityManagerInterface` (which is much more common).

In any case, thanks @JeroenMoonen for your contribution.

**Pull request merged**